pkg/servers/servers_test.go

```diff
@@ -27,6 +28,16 @@ var ready func(rw http.ResponseWriter, req *http.Request)
 var mux *http.ServeMux
 var httpClient, httpsClient *http.Client
 
+const RETRY_SECONDS = 30
+const RETRY_INTERVAL = 10 * time.Millisecond
```

Comment:
Nit, but I don't thing we generally define consts in caps and I think they should be named specifically to where they're used (for connection retry).